### PR TITLE
[ADD] Module which restore analytic account sequence

### PR DESCRIPTION
**Pull request by benwillig**

### CODE REVIEW

File: account_analytic_sequence/README.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (adrienpeiffer) [inline account_analytic_sequence/README.rst:1]: Can you adapt the readme file according the template https://github.com/OCA/maintainer-tools/tree/master/template/module

**Review by adrienpeiffer — APPROVED** [review]
:+1:

File: account_analytic_sequence/README.rst (hunk 1 above)
Comment (benwillig) [reply account_analytic_sequence/README.rst:1]: Done :alien:

**Review by ThomasBinsfeld — APPROVED** [review]
Functional and code review :+1:

File: account_analytic_sequence/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://www.acsone.eu',
+    'depends': [
+        'analytic',
+    ],
+    'data': [
+        'data/sequence.xml',
+    ],
+    'demo': [
```
Comment (rousseldenis) [inline account_analytic_sequence/__manifest__.py]: Avoid void sections

**Review by rousseldenis — CHANGES_REQUESTED** [review]
LGTM. Minor comment

File: account_analytic_sequence/__manifest__.py (hunk 2 above)
Comment (benwillig) [reply account_analytic_sequence/__manifest__.py]: Done!

**Review by rousseldenis — APPROVED** [review]

File: account_analytic_sequence/data/sequence.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+<odoo noupdate="1">
+
+    <record model="ir.sequence" id="analytic_account_code_seq">
+        <field name="name">Analytic account code</field>
+        <field name="code">account.analytic.account.code</field>
+        <field name="prefix">AA</field>
+        <field name="padding">3</field>
+    </record>
+
+</odoo>
```
Comment (LoisRForgeFlow) [inline account_analytic_sequence/data/sequence.xml]: end line

**Review by LoisRForgeFlow — APPROVED** [review]
Code LGTM, check minor comment in line

File: account_analytic_sequence/data/sequence.xml (hunk 3 above)
Comment (benwillig) [reply account_analytic_sequence/data/sequence.xml]: End line added. Thanks for the review!

**Comment by feketemihai** [comment]
@pedrobaeza this is approved and green ci

**Comment by pedrobaeza** [comment]
@aheficent this is what you were trying to implement the other day I think.

**Pull request merged**